app/containers/Forms/SummaryFormArrayFieldTemplate.tsx

```diff
@@ -3,28 +3,28 @@ import {ArrayFieldTemplateProps} from 'react-jsonschema-form';
 import {Form, Col} from 'react-bootstrap';
 
 const SummaryFormArrayFieldTemplate: React.FunctionComponent<ArrayFieldTemplateProps> = ({
-  TitleField,
-  title,
   items
 }) => {
   return (
     <>
-      <Col xs={12}>
-        <TitleField
-          id={null}
-          title={title === 'gases' || title === 'Source Types' ? null : title}
-          required={false}
-        />
-      </Col>
-      {items.map(element => {
+      {items.map((element, i) => {
         return (
           <React.Fragment key={element.index}>
```

Comment:
Can we not use `key=` on the `<>` fragment style?

Reply:
Nope